### DISCUSSION TITLE
Issue with Contributions Not Showing on My GitHub Profile

### DISCUSSION THREAD

**Discussion by wortellen** [community/community#147153] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello,

My name is Vadim Zelenevskii, and my GitHub username is wortellen. I am reaching out because I have an issue with contributions not appearing on my GitHub profile.

I have contributed to the [OpenStack Mistral](https://github.com/openstack/mistral) project, which is a mirrored repository from OpenDev. My commits are visible in the repository, and the author information correctly shows my email address (wortellen@gmail.com), which is verified in my GitHub account.

Examples of my commits:

[Add checksum field to wf definition](https://github.com/openstack/mistral/commit/ac8b2d0488452ea272d1b7947dd6ecb934a3fed6)
[Partial Workflow Failure Handling](https://github.com/openstack/mistral/commit/7cc0074d86d63b7b508f87763e1983b151204cd3)
However, these contributions are not reflected on my GitHub profile. I suspect the issue may be because this repository is a mirror.

Is there a way to ensure that these contributions are counted towards my GitHub profile, given that they are associated with my verified email address?

Thank you for your time and assistance.

Best regards,
Vadim Zelenevskii
wortellen@gmail.com

**Comment by mecodeatlas** [comment]
Hey @wortellen! Thanks for reaching out! 

To help you identify why your contributions aren’t showing up, I recommend checking out the documentation [Why are my contributions not showing up on my profile?](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/why-are-my-contributions-not-showing-up-on-my-profile). It provides a series of steps to troubleshoot this issue.

Additionally, we have a [discussion](https://github.com/orgs/community/discussions/66346) from another user with a very similar problem, which might give you some helpful insights to resolve this. :)

Best of luck!
